### PR TITLE
Cloud.gov worker process

### PR DESCRIPTION
**Pull request by rahearn**
**Description of change**

* Consolidate worker code back into `src` directory
* Update to cf-cli 7 to take advantage of `processes` definition in manifest.yml
* Add hses import test files to .cfignore to ensure we're not importing them instead of the proper files

**How to test**

Everything should work the same as before the change:

1) `yarn docker:start` 
2) Upload a file to a new AR, and ensure that the worker picks up the file and scans it for viruses.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/204

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [x] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,9 +75,9 @@ You may run into some issues running the docker commands on Windows:
 | `yarn docker:db:migrate:undo` | Undo migrations in docker containers | `yarn db:migrate:undo` | |
 | `yarn docker:db:seed` | Run all seeders located in `src/seeders` | `yarn db:seed` | |
 | `yarn docker:db:seed:undo` | Undo all seeders located in `src/seeders` | `yarn db:seed:undo` | |
-| | Install dependencies for the frontend and backend (for local development)  | `yarn deps:local` | |
```
Comment (rahearn) [inline README.md:78]: `yarn deps:local` is defined on another line as well

File: src/lib/updateGrantsGrantees.js [hunk 2]
```diff
@@ -90,9 +90,6 @@ export async function processFiles() {
   }
 }
 
-// reading archives
-const zip = new AdmZip('./hses.zip');
```
Comment (rahearn) [inline src/lib/updateGrantsGrantees.js:94]: I _think_ having this outside the function was leading to stale data being loaded. Moving it is part of my experiment to verify that or not, along with adding those files to `.cfignore`

**Review by dcmcand — APPROVED** [review]
Much cleaner. Nice job!

**Comment by rahearn** [comment]
```
└─> terraform apply

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # cloudfoundry_service_instance.database will be created
  + resource "cloudfoundry_service_instance" "database" {
      + id               = (known after apply)
      + name             = "ttahub-sandbox"
      + recursive_delete = false
      + service_plan     = "5ed9f319-4c75-4851-9885-598a96b7febc"
      + space            = "23d98919-d859-490d-bb30-b737915fbf48"
    }

  # cloudfoundry_service_instance.document_upload_bucket will be created
  + resource "cloudfoundry_service_instance" "document_upload_bucket" {
      + id               = (known after apply)
      + name             = "ttahub-document-upload-sandbox"
      + recursive_delete = false
      + service_plan     = "021bb2a3-7e11-4fc2-b06b-d9f5938cd806"
      + space            = "23d98919-d859-490d-bb30-b737915fbf48"
    }

  # cloudfoundry_service_instance.redis will be created
  + resource "cloudfoundry_service_instance" "redis" {
      + id               = (known after apply)
      + name             = "ttahub-redis-sandbox"
      + recursive_delete = false
      + service_plan     = "39e035d8-f847-4bf5-a2ee-521c3118cc42"
      + space            = "23d98919-d859-490d-bb30-b737915fbf48"
    }

Plan: 3 to add, 0 to change, 0 to destroy.
...
Apply complete! Resources: 3 added, 0 changed, 0 destroyed.
```

**Comment by rahearn** [comment]
Dev:
```
└─> terraform apply
cloudfoundry_service_instance.document_upload_bucket: Refreshing state... [id=5d4ab5b7-fa4e-4bb3-bca9-ea317ac6351e]

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # cloudfoundry_service_instance.database will be created
  + resource "cloudfoundry_service_instance" "database" {
      + id               = (known after apply)
      + name             = "ttahub-dev"
      + recursive_delete = false
      + service_plan     = "5ed9f319-4c75-4851-9885-598a96b7febc"
      + space            = "26178bf7-6275-45c5-9972-35ed819cf7c9"
    }

  # cloudfoundry_service_instance.redis will be created
  + resource "cloudfoundry_service_instance" "redis" {
      + id               = (known after apply)
      + name             = "ttahub-redis-dev"
      + recursive_delete = false
      + service_plan     = "39e035d8-f847-4bf5-a2ee-521c3118cc42"
      + space            = "26178bf7-6275-45c5-9972-35ed819cf7c9"
    }

Plan: 2 to add, 0 to change, 0 to destroy.
...
Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
```

**Pull request merged**